mypy.ini

```diff
@@ -9,4 +9,7 @@ ignore_missing_imports = True
 ignore_missing_imports = True
 
 [mypy-ifaddr]
+ignore_missing_imports = True
+
+[mypy-fsoe_master.*]
 ignore_missing_imports = True
```

Comment:
We could add a typed marker, although the project cannot have mypy....

Reply:
Yes, I think we should just ignore it.